src/components/FileUpload/FileUpload.js

```diff
@@ -0,0 +1,176 @@
+import React from 'react';
+import Dropzone from 'react-dropzone';
+import PropTypes from 'prop-types';
+
+import uploadArrow from './upload_arrow.png';
+import complete from './complete.png';
+
+import styled from 'styled-components';
+
+import ProgressBar from './ProgressBar';
+import Card from './Card';
+import LinkButton from './LinkButton';
+import classnames from 'classnames';
+
+const UploadImg = styled.img`
+  width: 85px;
+  display: block;
+  margin: 10px auto;
+`;
+
+const P = styled.p`
+  margin: 0;
+  text-align: center;
+`;
+
+const ActionMessage = styled(P)`
+  font-weight: bold;
+`;
+
+const ErrorMessage = styled(P)`
+  font-size: 13px;
+  color: #ff3b3b;
+`;
+const ManualMessage = styled(P)`
+  color: #aaaaaa;
+  display: inline;
+`;
+
+const DropZoneFile = styled(Dropzone)`
+  display: inline;
+  color: #3b86ff;
+  text-decoration: underline;
+  cursor: pointer;
+`;
+
+const CenterDiv = styled.div`
+  text-align: center;
+`;
+
+const StyledCard = styled(Card)`
+  height: 150px;
+  &.disabled {
```

Comment:
Shouldn't this just be in Card when state is `disabled`? I can't see you using that state anywhere